source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -3300,6 +3311,10 @@ export class Container extends CardElementContainer {
     private _renderedItems: Array<CardElement> = [];
     private _style?: string = null;
 
+    public getItems(): Array<CardElement> {
```

Comment:
We can't add this, please remove.  The _items field isn't supposed to be accessible. Developers shouldn't access it directly as it breaks the mechanism implemented by addItem() that ensures an element's parent is properly set.

Reply:
Already in progress

Reply:
In progress.